graceful/service.go

```diff
@@ -9,15 +9,16 @@ import (
 	"sync"
 	"time"
 
-	"github.com/facebookgo/grace/gracenet"
-	"gopkg.in/errgo.v1"
+	"github.com/Scalingo/go-utils/errors/v2"
+
+	"github.com/cloudflare/tableflip"
 
 	"github.com/Scalingo/go-utils/logger"
 )
 
-type service struct {
+type Service struct {
```

Comment:
I've exported the "Service" struct. I'm not sure if you will agree with this, but it is generally better practice as it maintains type safety. 
Another option would be for NewService to return an interface, we could then keep the service struct private.
I did not think this would be a dangerous change, as there are no exported fields within the struct.

Reply:
Actually I'm good with it, returning a private entity from a public package is not a good practice, indeed a public struct or an interface would do. Considering this a good improvement :+1: